common/lib/xmodule/xmodule/video_module/video_handlers.py

```diff
@@ -276,7 +276,7 @@ def transcript(self, request, dispatch):
 
         elif dispatch.startswith('available_translations'):
 
-            available_translations = self.available_translations(transcripts)
+            available_translations = self.available_translations(transcripts, verify_assets=True)
```

Comment:
In this specific line, I'm being totally backward compatible. So, it used to verify in `master` (so to speak), and I'd like my PR to do the same here.

The other option, where `master` used to skip verification, this PR allows enforcing verifications in those cases.

Old Logic -> New Logic:
 - Verified (by default with function signature) -> Verified (enforced by the callee) = Logic haven't changed, backward compatible and hard-coded.
 - Not verified (by callee) -> Configurable (by feature flag)

I hope that explains it